sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keys/key_type.hpp

```diff
@@ -12,19 +13,48 @@
 
 namespace Azure { namespace Security { namespace KeyVault { namespace Keys {
 
-  enum class KeyTypeEnum
+  /**
+   * @brief The JsonWebKey types.
+   *
+   */
+  enum class Kty
```

Comment:
Kty is not very descriptive and likely to collide as well.

Reply:
updated to `JsonWebKeyType`